arm64-gen.c

```diff
@@ -1992,6 +1992,127 @@ ST_FUNC void gen_vla_alloc(CType *type, int align) {
     vpop();
 }
 
+#ifdef HAVE_PTRAUTH
+ST_FUNC void gen_ptrauth_strip_i(void) {
+    CType type;
+    uint32_t r_dst, r_src;
+
+    type = vtop->type;
+    r_src = intr(gv(RC_INT));
+    vpop();
+
+    r_dst = get_reg(RC_INT);
+
+    o(0xaa0003e0 | r_dst | r_src << 16); // mov x(r_dst), x(r_src)
+    o(0xdac143e0 | r_dst); // xpaci x(r_dst)
+
+    vpushi(0);
+    vtop->r = r_dst;
+    vtop->type = type;
+}
+
+ST_FUNC void gen_ptrauth_strip_d(void)
+{
```

Comment:
(This and the functions below also need the same style fix.)

Reply:
Ugh, the coding style in TinyCC is somewhat inconsistent as I also noticed the other style above here. I guess we should stick to what the oldest code is doing (the code written by Fabrice).

Reply:
ok let me check

Reply:
idk, oldest commits are very weird, the style is like this:

- no types on args and usage of empty arg list to mean "anything"
- curly braces on new line on function definitions, un the same line for all the rest

```
int oad(c, s)
{
    o(c);
    *(int *)ind = s;
    s = ind;
    ind = ind + 4;
    return s;
}

/* push to value stack */
void vset(t, v)
{
    vt = t;
    vc = v;
}

/* generate a value in eax from vt and vc */
void gv()
{
#ifndef TINY
    int t;
#endif
    if (vt & VT_LVAL) {
        if ((vt & VT_TYPE) == VT_BYTE)
            o(0xbe0f);   /* movsbl x, %eax */
        else
            o(0x8b);     /* movl x,%eax */
        if (vt & VT_CONST)
            oad(0x05, vc);
        else if (vt & VT_LOCAL)
            oad(0x85, vc);
        else
            g(0x00);
    } else {
        if (vt & VT_CONST) {
            oad(0xb8, vc); /* mov $xx, %eax */
        } else if (vt & VT_LOCAL) {
            oad(0x858d, vc); /* lea xxx(%ebp), %eax */
        } else if (vt & VT_CMP) {
...
```

Reply:
Ah wow, that looks like really old C :)